### ISSUE TITLE
Why would volatility be a problem for arbitraging?

### ISSUE THREAD

**Issue by adrianog** [kelvinau/crypto-arbitrage#16]
Hi

Thanks for the useful resource.

Question:
You seem to imply that high volatility is not conducive to fruitful arbitraging, whereas my understanding was the opposite - that the volatility is creating the arbitrage opportunities.
Would you mind expanding on the topic?

Many thanks

**Comment by kelvinau** [comment]
That is true, in some way. High volatility creates a lot of arbitrage opportunities to an extent. When there's no limit how much the price can go up or down, that's a different story. Take bitcoin as an example. It reached $19k in December and dropped to $12k in two weeks. Let's say you had 1 BTC (worth $19k in Dec.) to begin with and you spotted a lot of arbitrage opportunities. You made 0.5 BTC in two weeks, so at the end you would have 1.5 BTC, but that would be worth only $18k ($12k * 1.5). What did you earn? You actually lost $1k because of the huge price dropping. Of course if you made more BTC, you would probably be still earning, but just making less profit - only if there were a lot of opportunities. Some can say that you should never hold BTC, or should convert some to fiat currency immediately or regularly, but that will be another discussion of fiat-to-crypto conversion.

Now it's relatively more stable than in December. I think It could be a good time to re-look at the opportunities. As always, trade at your own risk.

**Comment by adrianog** [comment]
You had 1 BTC to start with, so the arbitrage increased on that. Yes, whether and when to take money in / out to fiat is another matter. If you are bullish on crypto medium/long term (as I am) you can keep everything in crypto all the time and arbitrage crypto/crypto pairs.

Another unrelated comment hijacking the thread if you don't mind.
Rebalancing your (err..) balances. Token balances at the exchanges. It might be worth executing smaller arbitrage trades (before larger ones), even "no loss no gain" trades, and even small loss trades if there is a need to rebalance funds that have become skewed, in a particular exchange. To maintain a uniform set of balances (with only maybe ETH as an exception) at each exchange, as having a balance in a particular token is equivalent with being able to take an opportunity on that particular token, and therefore it contributes to creating a larger order book universe - and a greater order book universe means more arbitrage opportunities.

I'm not too sure whether in practice that is indeed possible (i.e. no opportunity to rebalance meaningfully - as it will wipe out arbitrage gains if you try to do that, there is not waiting around that does it, brining the money back into a balance will always lose you too much / more than the initial arbitrage gain), or if one has to necessarily go the cumbersome "send the fiat/ETH/transportable_token out "manually" to the other exchange (wallet sends, withdrawal, / deposits of fiat etc)" route.
